### PR TITLE
centralize all the prettier/eslint config

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ Drupal and the React app. If you want to test out your changes in this context,
 `drupal-admin-ui` directory and run `yarn workspace admin-ui build`. You can then visit the URL that
 `composer start` generated.
 
+Further helpful commands are `yarn run eslint` and `yarn run prettier`.
```
Comment (justafish) [inline README.md]: I don't think you need `run`

File: package.json [hunk 2]
```diff
@@ -11,6 +11,9 @@
     "packages/*"
   ],
   "scripts": {
-    "test": "yarn workspace admin-ui test"
+    "eslint": "eslint packages/*/src --ext .js,.jsx,.ts,.tsx  --max-warnings=0 --fix",
```
Comment (justafish) [inline package.json]: nice :ok_hand:

File: README.md (hunk 1 above)
Comment (dawehner) [reply README.md]: Removed "run"

**Comment by dawehner** [comment]
@justafish Do you think your review is addressed now?

**Pull request merged**